src/utils/network.ts

```diff
@@ -143,8 +143,8 @@ export const releaseBlockRequest = (id: string, note: string): AxiosPromise =>
             data: {
                 note,
             },
-            method: "post",
-            url: `${config.blockRegistry.host}/blocks/${id}`,
+            method: "put",
```

Comment:
are we using lower case in other parts of the codebase?

Reply:
yea, but I can change to uppercase if you wish.

Reply:
Seems like everyone uses it uppercase, but it's up to you.

Reply:
We shall go uppercase :)